README.md

```diff
@@ -1,21 +1,25 @@
 # Project Name
 
-> One paragraph statement about the project.
+> A SET OF COLORED MARKERS.
```

Comment:
can you change it to lower case? And the paragraph is for us to describe what the project entails and not just topic